wolfssl/internal.h

```diff
@@ -146,14 +146,14 @@
         /* do nothing */
 #elif defined(WOLFSSL_CMSIS_RTOS)
     #include "cmsis_os.h"
+#elif defined(WOLFSSL_CMSIS_RTOSv2)
+    #include "cmsis_os2.h"
+#elif defined(WOLFSSL_CMSIS_RTOSv2)
```

Comment:
There are two identical WOLFSSL_CMSIS_RTOSv2 blocks here.  Please remove one.

Reply:
Removed the duplication.

Reply:
Thanks